### DISCUSSION TITLE
Why nest an unordered list need 2 leading spaces, but 3 is need for ordered list

### DISCUSSION THREAD

**Discussion by Valen-Pony** [remarkjs/remark#825] [Q&A] [answered]
text:
```md
- unordered
  - 2 space

- unordered
  1. 2 space

- unordered
   - 3 space

- unordered
   1. 3 space

1. ordered
  1. 2 spaces

1. ordered
  - 2 spaces

1. ordered
   1. 3 spaces

1. ordered
   - 3 spaces
```

![md](https://user-images.githubusercontent.com/83207497/131076048-7f9f5bca-e874-4d6a-8f0d-6c0492150cff.png)

**Comment by ChristianMurphy** [comment, marked as answer]
Short answer, in a monospace font it's the number of spaces for it to visually be inside the other list.

For a more comprehensive answer see the spec https://spec.commonmark.org/0.30/#example-312

And the CommonMark team is also fairly responsive to clarifying questions on the spec at https://talk.commonmark.org/

**Reply by Valen-Pony** [reply]
Aha, got it. `- ` are 2 charactors and `1. ` are 3 charactors. Thanks for the reply and such wonderful lib!

**Reply by Valen-Pony** [reply]
One more question, is it standard or can be expected that I can write 3 spaces to nested under an unordered list? I want to tell my editor users just add 3 spaces to nested a list, regardless the types

**Reply by wooorm** [reply]
I personally always use an even 4!
